scripts/profileImageUrlUpdater.js

```diff
@@ -0,0 +1,35 @@
+// For Issue #173
```

Comment:
```suggestion
// Issue #173을 해결하기 위한 DB 마이그레이션 스크립트입니다.
// https://github.com/sparcs-kaist/taxi-back/issues/173
```
이렇게 의미 있는 스크립트 설명을 추가해주시면 좋습니다!